Resources/Private/Form/Partials/Form/Navigation.html

```diff
@@ -16,7 +16,7 @@
 					<f:form.button name="__currentPage" value="{form.pages -> f:count()}" class="btn btn-primary">
 						<f:if condition="{form.renderingOptions.submitButtonLabel}">
 							<f:then>
-								{form.renderingOptions.submitButtonLabel -> f:translate(package: '{form.renderingOptions.translationPackage}')}
+								{f:translate(id: 'forms.navigation.{form.identifier}.submit', package: '{form.renderingOptions.translationPackage}', value: form.renderingOptions.submitButtonLabel)}
```

Comment:
Why is the form identifier part of the l10n id now? in the other labels that's not the case, for example `forms.navigation.nextPage`